src/BenchmarksClient/Startup.cs

```diff
@@ -96,8 +96,13 @@ public static int Main(string[] args)
                 var job = allJobs.FirstOrDefault();
                 if (job != null)
                 {
-                    var jobLogText = $"[ID:{job.Id} Connections:{job.Connections} Threads:{job.Threads} " +
-                        $"Duration:{job.Duration} Pipeline:{job.PipelineDepth} Method:{job.Method}";
+                    var jobLogText =
+                        $"[ID:{job.Id} Connections:{job.Connections} Threads:{job.Threads} Duration:{job.Duration} Method:{job.Method}";
+
+                    if (!string.IsNullOrEmpty(job.ScriptName) && job.PipelineDepth > 0)
```

Comment:
> Both existing scripts already do that.

The scripts test that `args[1] ~= nil`. In Lua 0 != nil, so passing 0 would set the pipeline depth to 0.